packages/block-library/src/paragraph/edit.js

```diff
@@ -108,28 +109,31 @@ function ParagraphBlock( {
 		} ),
 		style: { direction },
 	} );
+	const blockEditingMode = useBlockEditingMode();
 
 	return (
 		<>
-			<BlockControls group="block">
-				<AlignmentControl
-					value={ align }
-					onChange={ ( newAlign ) =>
-						setAttributes( {
-							align: newAlign,
-							dropCap: hasDropCapDisabled( newAlign )
-								? false
-								: dropCap,
-						} )
-					}
-				/>
-				<ParagraphRTLControl
-					direction={ direction }
-					setDirection={ ( newDirection ) =>
-						setAttributes( { direction: newDirection } )
-					}
-				/>
-			</BlockControls>
+			{ blockEditingMode === 'default' && (
+				<BlockControls group="block">
+					<AlignmentControl
+						value={ align }
+						onChange={ ( newAlign ) =>
+							setAttributes( {
+								align: newAlign,
+								dropCap: hasDropCapDisabled( newAlign )
+									? false
+									: dropCap,
+							} )
+						}
+					/>
+					<ParagraphRTLControl
```

Comment:
Why was this moved in the condition?

Reply:
Why not? It isn't something that should appear for `contentOnly` or `disabled` blocks.

Reply:
It's a language thing no? That's content?